tests/test_user/test_change_user_role.py

```diff
@@ -18,7 +18,7 @@ def test_change_user_role(self):
         """
         Testing change of user role
         """
-        CommonTestCases.admin_token_assert_equal(
+        CommonTestCases.admin_token_assert_in(
```

Comment:
Consider adding test for changing already existing user role.

Reply:
Done